### PR TITLE
Mitaka and newton compatibility

### PR DESCRIPTION
**Pull request by AlvaroVega**
fixes https://github.com/telefonicaid/fiware-keystone-scim/issues/43

### CODE REVIEW

File: keystone_scim/contrib/scim/controllers.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
```
Comment (fgalan) [inline keystone_scim/contrib/scim/controllers.py]: Does this PR needs changelog entry?

File: keystone_scim/contrib/scim/controllers.py [hunk 2]
```diff
@@ -20,7 +20,7 @@
 
 """Extensions supporting SCIM."""
 
-from keystone import config
+#from keystone import config
```
Comment (fgalan) [inline keystone_scim/contrib/scim/controllers.py]: Leftover comment. Should be removed

File: ChangeLog [hunk 3]
```diff
@@ -1,6 +1,10 @@
 CHANGES
 =======
 
+v1.1.8
+-------
+Upgrade plugin for compatibility with Mitaka version (including docker) [#43]
```
Comment (fgalan) [inline ChangeLog]: Should Netwon be mentioned also in this line?

File: keystone_scim/contrib/scim/controllers.py (hunk 2 above)
Comment (AlvaroVega) [reply keystone_scim/contrib/scim/controllers.py]: Yes, It was already removed.

File: ChangeLog (hunk 3 above)
Comment (AlvaroVega) [reply ChangeLog]: updated

**Comment by fgalan** [comment]
LGTM

**Pull request merged**